responses/__init__.py

```diff
@@ -601,7 +620,35 @@ def add_passthru(self, prefix):
         """
         if not isinstance(prefix, Pattern) and _has_unicode(prefix):
             prefix = _clean_unicode(prefix)
-        self.passthru_prefixes += (prefix,)
+        self._passthru_prefixes += (prefix,)
+        response = PassthroughResponse(method, prefix)
+        response.check_fired = check_fired
+        response.priority = Priority.LOW
+        self.add(response)
+
+    @property
+    def passthru_prefixes(self):
+        warn(
+            "Direct access of passthru_prefixes is depricated. "
+            "Passthroughs are stored as PassthroughResponses.",
+            DeprecationWarning,
+        )
+        return self._passthru_prefixes
+
+    @passthru_prefixes.setter
+    def passthru_prefixes(self, prefixes):
+        warn(
+            "Setting passthru_prefixes directly is depricated. Please use add_passthru",
```

Comment:
Again, don't know why this is necessary.

Reply:
See above.